assets/stylesheets/landing.css

```diff
@@ -32,25 +43,35 @@ body {
 }
 
 .glitch {
-  font-size: 18vh;
+  color: #f8f8f8;
+  font-size: 15vh;
+  font-family: 'octopost';
+  text-shadow: 0px 8px 13px rgba(0, 0, 0, 0.2),
+  0px 18px 23px rgba(0, 0, 0, 0.3);
 }
 
 .main-button {
-  position: fixed;
-  top: 90vh;
-  left: 50%;
-  transform: translate(-50%, -50%);
+  position: absolute;
+  top: 88%;
+  left: 43vw;
```

Comment:
you can keep it centered horizontally, as it was kept before using `left:50%` and `transform:translateX(-50%)`, 43vw is really experimental and not necessarily centered.

And instead of `top: 88%`, so random unit!! use `bottom: ??rem/px`, 4rem will do, I guess.